assets/js/atomic/components/product/price/index.js

```diff
@@ -33,12 +34,18 @@ const ProductPrice = ( { className, product } ) => {
 					className={ `${ layoutStyleClassPrefix }__product-price__value` }
 				>
 					<NumberFormat
-						value={ prices.price_range.min_amount }
+						value={
+							prices.price_range.min_amount /
+							10 ** prices.currency_minor_unit
```

Comment:
`x / 10 ** prices.currency_minor_unit` seems to be used in several places in this and other files of this PR. I was wondering if it might make sense extracting it to a util function? I don't have a strong opinion about this so mostly asking what you think, but I find operations with exponents are a bit hard to rationale if you are not used to the code, so I think encapsulating it into its own function might make the code easier to read.

Reply:
Another advantage of moving things into the `FormattedPrice` component is that it limits the surface area of `NumberFormat` in case we decide down the road that we want to switch to another library because of size or compatibility issues.

Reply:
Instead of `FormattedPrice` it might be better to call it `FormattedMoney` as that's a bit more broader description of it's purpose. That would leave the door open for having various other implementations of `NumberFormat` for other things like `FormattedPhoneNumber` or `FormattedCreditCard` etc (which `NumberFormat` seems to be great for).